tests/test_query.py

```diff
@@ -33,24 +34,27 @@ def test_population_mask():
 
 
 def test_resolve_ids():
-    data = pd.DataFrame([
-        [1, .4, 'seven'],
-        [2, .5, 'eight'],
-        [3, .6, 'nine']],
-        columns=['int', 'float', 'str'])
-    assert [False, True, False] == resolve_ids(data, '', {'str': 'eight'}).tolist()
-    assert [False, False, False] == resolve_ids(data, '', {'int': 1, 'str': 'eight'}).tolist()
+    data = pd.DataFrame(
+        [[1, 0.4, "seven"], [2, 0.5, "eight"], [3, 0.6, "nine"]], columns=["int", "float", "str"]
```

Comment:
Minor, in this case adding a trailing `,` after `[3, 0.6, "nine"]` would preserve the separate lines and it seems a bit clearer IMHO.